atlasdb-backup/src/test/java/com/palantir/atlasdb/backup/AtlasBackupServiceTest.java

```diff
@@ -98,17 +98,19 @@ public void prepareBackupRegistersLockForRefresh() {
     @Test
     public void prepareBackupThrowsIfNamespacesCollide() {
         AtlasService collidingAtlasService = AtlasService.of(ServiceId.of("c"), ATLAS_SERVICE.getNamespace());
-        assertThatThrownBy(
+        assertThatLoggableExceptionThrownBy(
                         () -> atlasBackupService.prepareBackup(ImmutableSet.of(ATLAS_SERVICE, collidingAtlasService)))
-                .isInstanceOf(SafeIllegalArgumentException.class);
+                .isInstanceOf(SafeIllegalArgumentException.class)
+                .hasMessageContaining("Duplicated namespaces");
```

Comment:
nit: I think there is just `hasLogMessage`, and you can even assert on the safe args